### DISCUSSION TITLE
delay app updates

### DISCUSSION THREAD

**Discussion by ghost** [nextcloud/all-in-one#848] [Ideas] [answered]
Possibility to delay app updates by one week. Because app updates often cause bugs that are fixed in subsequent updates.

(I would open a discussion but unfortunately no longer works)

**Comment by szaimen** [comment]
> (I would open a discussion but unfortunately no longer works)

I'll move it to discussions for you👍

**Comment by szaimen** [comment]
Honestly in such cases just restore the backup or disable automatic updates which will come with https://github.com/nextcloud/all-in-one/issues/722

**Comment by ghost** [comment]
For over a week there are bugs in the calendar app and with the last update more bugs have been added. The whole thing is unfavorable for a productive system and could be avoided in the future with the app update delay.

**Reply by szaimen** [reply]
We will allow to disable automatic updates with https://github.com/nextcloud/all-in-one/issues/722 as I said above

**Comment by szaimen** [comment, marked as answer]
Will be addressed with https://github.com/nextcloud/all-in-one/pull/897

**Reply by szaimen** [reply]
This is now released with 1.6.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
